packages/fxa-shared/metrics/glean/fxa-ui-metrics.yaml

```diff
@@ -1386,7 +1386,43 @@ cad:
     expires: never
     data_sensitivity:
       - interaction
-
+third_party_auth:
+  apple_deeplink:
+    type: event
+    description: |
+      User sees Apple link and bypasses Mozilla Accounts email first/login
```

Comment:
nit: What about this "User clicked Apple/Google Signin link from an RP hosted site and is taken directly to Apple/Google authentication flow, bypassing Mozilla Accounts email first page."

Reply:
SGTM, thanks!  I copied that from the Google Sheet.  😅